Makefile

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2019 King's College London
+# Created by the Software Development Team <http://soft-dev.org/>
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# http://www.apache.org/licenses/LICENSE-2.0>, or the MIT license <LICENSE-MIT
+# or http://opensource.org/licenses/MIT>, or the UPL-1.0 license
+# <http://opensource.org/licenses/UPL> at your option. This file may not be
+# copied, modified, or distributed except according to those terms.
+
+FETCH=wget
+
+RENAISSANCE_V=0.9.0
+RENAISSANCE_JAR=renaissance-gpl-${RENAISSANCE_V}.jar
+
+GRAALCE_V=1.0.0-rc16
+GRAALCE_DIR=graalvm-ce-${GRAALCE_V}
+GRAALCE_TGZ=graalvm-ce-${GRAALCE_V}-linux-amd64.tar.gz
+
+all: setup
```

Comment:
This line feels redundant: if "all" this `Makefile` will ever do is build things, then we might as well have `setup` as the first target.

Reply:
Good point. Just `all` is a convention. I don't feel super strongly.

Reply:
I think `all` is useful if it calls multiple targets; in this instance it feels a bit convention-for-the-sake-of-it TBH.

Reply:
Fair enough. I'll kill it. If another target comes along that we want to run by default then we can bring it back.